File: PHP/card.php

<?php
    require_once('payment.php');
    class Card extends Payment{
        public $cardNumber;
        public $cardDate;
        public $CVC;

        public function __construct($id,$cardNumber,$cardDate,$CVC){
            parent::__construct($id);
            $this->cardNumber = $cardNumber;
            $this->cardDate = $cardDate;
            $this->CVC = $CVC;
        }        
    }

?>

File: Python/card.py
from payment import Payment

class card(Payment):
    cardNumber = int
    cardDate = int
    CVC = str

    def __init__(self, id, cardNumber, cardDate, CVC):
        super().__init__(id)
        self.cardNumber = cardNumber
        self.cardDate = cardDate
        self.CVC = CVC
